frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -128,7 +128,10 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
 
         return <section className="entity-details">
             <EntityNavigation
-                entity={ state.selectedEntity }
+                nextEntity={ state.nextEntity }
+                previousEntity={ state.previousEntity }
+                dispatch={ this.props.dispatch }
```

Comment:
`dispatch` shouldn't be used outside of connected containers (like this one). Instead you should pass a method that will, in turn, use `dispatch`. This way you can also remove the `router` attribute.